allensdk/brain_observatory/behavior/data_objects/rewards.py

```diff
@@ -26,6 +27,14 @@ def from_stimulus_file(
         """Get reward data from pkl file, based on timestamps
         (not sync file).
         """
+
+        if not np.isclose(stimulus_timestamps.monitor_delay, 0.0):
```

Comment:
Instead of doing this check, why not just call `subtract_monitor_delay` and issue a warning?

Reply:
This is just personal preference. Warnings worry me. I feel as though they are too easily missed in all of the other noise that often gets written to stdout. If you want to revisit this, we can. I want to get this merged soon so that we can produce the first batch of NWB files for validation.